defender/staged/downloaded/Forta_Oracle_Price.js

```diff
@@ -133,17 +133,23 @@ exports.handler = async function (autotaskEvent) {
   // extract the metadata
   const {
     underlyingTokenAddress,
+    protocolVersion,
   } = metadata;
   if (underlyingTokenAddress === undefined) {
     throw new Error('underlyingTokenAddress undefined');
   }
 
+  // Handle older alerts which don't specify the protocol version
+  let versionString = '';
+  if (protocolVersion !== undefined) {
+    versionString = ` (Compound v${protocolVersion})`;
+  }
```

Comment:
This can be hardcoded to (Compound v2). The Oracle Price Bot checks for an event that is only emitted on Compound V2 contracts.